src/tools/setup-virtualenv.sh

```diff
@@ -17,6 +17,11 @@
 
 SCRIPTNAME="$(basename $0)"
 PYTHON_BINARY="python2.7"
+if [ `uname` == FreeBSD ]; then
+    GETOPT=/usr/local/bin/getopt
+else
+    GETOPT=getopt
```

Comment:
put `getopt` in double-quotes? I know it's not striclty necessary - just to unify the coding style.

Reply:
@smithfarm 
Sure, no problem.